js/file.js

```diff
@@ -0,0 +1,18 @@
+function showCloseMenu() {
+  const menuSection = document.querySelector('#mobile-nav');
+  const nav = document.querySelector('#desktop-nav');
+  const headerName = document.querySelector('#logo');
+  const itemsContainer = document.querySelector('#items-container-menu');
+
+  menuSection.classList.toggle('d-none');
+  nav.classList.toggle('d-none');
+  headerName.classList.toggle('d-none');
+  headerName.classList.toggle('d-flex-center');
+  itemsContainer.classList.toggle('d-none');
+  itemsContainer.classList.toggle('d-flex');
```

Comment:
- [ ] [OPTIONAL] The logo and hamburger menu still shows on your mobile menu wherein the design they should not show let's make sure we follow the design 
 
Your mobile menu | Figma design
:----:|:-----:
![image](https://user-images.githubusercontent.com/87689547/147375923-4f7df584-b8a7-49fe-9ba6-3043f4e5c62d.png) | ![image](https://user-images.githubusercontent.com/87689547/147375955-1fce3f46-7ae4-4f60-842f-f0d68c8af031.png)